apps/dashboard/src/components/workflow-editor/steps/conditions/edit-step-conditions-form.tsx

```diff
@@ -102,6 +102,9 @@ export const EditStepConditionsForm = () => {
     const updateStepData: Partial<StepUpdateDto> = {
       controlValues: { ...step.controls.values, skip },
     };
+    if (typeof skip === 'boolean') {
+      updateStepData.controlValues!.skip = null;
+    }
```

Comment:
why is this needed?

Reply:
didn't update with latest next, will do that